smartsim/_core/control/controller.py

```diff
@@ -135,6 +135,10 @@ def start(
             # it may be called seperately
             self.poll(5, True, kill_on_interrupt=kill_on_interrupt)
 
+    @property
+    def active_orch_list(self) -> t.Dict[str, Job]:
+        return self._jobs.db_jobs
```

Comment:
I think we might introduce a way for external code to mess with your orchestrator list here by returning `db_jobs` directly.

Can you write a test that uses this property, removes something from the list, and shows that `self._jobs.db_jobs` remains unchanged? 

Probably solvable w/something as simple as `return {}.update(self._jobs.db_jobs)`

Reply:
changed to return {**self._jobs.db_jobs}, and added test